install_msw.sh

```diff
@@ -127,6 +133,38 @@ then
     git config --global core.fileMode false
 fi
 
+if [ "WSL" = "$platform" ]
+then
+    # Install/update packages.
+    packages_list="autoconf automake bsdtar dos2unix doxygen \
```

Comment:
Prefer to use single quotes when there is nothing to expand inside the string.

Reply:
I keep it as is because of shellcheck.